### DISCUSSION TITLE
failed to mkdir "/etc/pwx": mkdir /etc/pwx: read-only file system

### DISCUSSION THREAD

**Discussion by paracetamol32** [siderolabs/talos#6948] [Q&A]
Hi, I try to use portworx essential with talos, but px-cluster pods want write access to /etc  (immutable in talks if I understand).
I have this error:
 failed to mkdir "/etc/pwx": mkdir /etc/pwx: read-only file system
Have you a workaround to change that ?
Thank you !!!

**Comment by smira** [comment]
Most of Talos `rootfs` is read-only. The workaround depends on the nature of the workload. You can stop mounting `/etc` from the host (so it stays within the container), or move it to use path from `/var`.

**Comment by paracetamol32** [comment]
I think it is the case. storagecluster is configured like this: 
  volumes:
    - name: etcpwx
      hostPath:
        path: /var/pwx
        type: ''
   volumeMounts:
      - name: etcpwx
        mountPath: /etc/pwx
